docs/2.0/docs/patcher/tutorials/applying-first-patch.md

```diff
@@ -1,3 +1,163 @@
+<summary>Hitting `u` to view changes shows the following:</summary>
+
+![The changes page shows every unit that uses the module, and what the most recent version it](/img/patcher/tutorials/patcher_update_usages.png)
+</details>
+
+<details>
+<summary>Hitting `v` to view the changelog shows the following:</summary>
+
+![The changelog page shows the changelog directly from the upstream module](/img/patcher/tutorials/patcher_update_changelog.png)
+</details>
+
+From here hit `enter` and patcher will apply the update:
+
+![Patcher update shows that it found a patch to be applied and what version it is incrementing to](/img/patcher/tutorials/patcher_update_in_progress.png)
+
+![Patcher Update completion notice](/img/patcher/tutorials/patcher_update_complete.png)
+
+<!-- spell-checker: disable -->
+You can now inspect the filesystem and see the results of the patch.  In this case it resulted in changing the version of `patcher-test` from `v0.10.3` to `v0.10.5` and added a new required argument `sampleinput` to the unit. This change can now be committed and pushed to your repository.
+<!-- spell-checker: enable -->
+
+![Patcher Update completion notice](/img/patcher/tutorials/patcher_update_results.png)
+
+## Applying a patch using Patcher via GitHub Actions
+
+Choose a directory named: <CustomizableValue id="DIRECTORY" /> in your repository where we'll add a new infrastructure unit.  In that folder create a file called `terragrunt.hcl` with the following content:
+
+```hcl title="$$DIRECTORY$$/terragrunt.hcl"
+
+terraform {
+  source = "git@github.com:gruntwork-io/terraform-aws-utilities.git//modules/patcher-test?ref=v0.10.3"
+}
+
+# Include the root `terragrunt.hcl` configuration, which has settings common across all environments & components.
+include "root" {
+  path = find_in_parent_folders()
+}
+```
+
+Now we'll push this up to your repository.
+
+```bash
+$ git checkout -b patcher-test-update
+$ git add $$DIRECTORY$$/terragrunt.hcl
+$ git commit -m "Adding a new infrastructure unit that uses the patcher-test module"
+$ git push -u origin patcher-test-update
+
+```
+
+Now navigate to your repository in GitHub and create a new pull request for the `patcher-test-update` branch.  In order for Patcher to identify this unit as needing an update, it first has to be merged to main, so go ahead and merge the pull request.
+
+:::info
+In a real world use case, you are unlikely to create a new unit that immediately is out of date, and instead Patcher would on a schedule and detect updates to modules some time after they have been merged.  For the purpose of this tutorial, however, we're intentionally merging an out of date module to main so we can trigger Patcher to update it immediately.
+:::
+
+Once the code has been merged to main, we can now directly trigger Patcher to scan for out of date modules.
+
+<details>
+<summary>If you don't already have Patcher installed in your repository, you can add the following GitHub Actions workflow:</summary>
```

Comment:
just a q. is it important to note somewhere in this doc that patcher is a paid tool? i don't know that i have a strong opinion, but the way this sentence is phrased feels a little bit like "oh, i could just do this and see immediate results" regardless of whether or not i'm a customer

Reply:
oh, never mind. i see it at the very end as a note. leaving the comment up in case other folks have that same question, but i see where it's included now!

Reply:
That's likely a true statement about a lot of our docs pages - there's an implicit assumption you're a customer and have access to the portal / tokens that provide access to gruntwork.  I think we can do better, but not necessarily required for this one page.